connect-examples/v2/csharp_payment/Pages/ProcessPayment.cshtml.cs

```diff
@@ -54,15 +46,15 @@ public static string Charge(string nonce)
             try
             {
                 var response = transactionsApi.Charge(LocationId, body);
-                return "Transaction complete\n" + response.ToJson();
+                this.ResultMessage = "Transaction complete!   " + response.ToJson();
             }
             catch (ApiException e)
             {
-                return e.Message;
+                this.ResultMessage = e.Message;
             }
         }
 
-        public static string NewIdempotencyKey()
+        private static string NewIdempotencyKey()
```

Comment:
Where is this method called?

Reply:
In `OnPost()`: `string uuid = NewIdempotencyKey();`